code/game/objects/items/devices/radio/radio.dm

```diff
@@ -66,8 +66,6 @@ var/global/list/default_medbay_channels = list(
 /obj/item/device/radio/New()
 	..()
 	wires = new(src)
-	if(radio_controller)
-		initialize()

```

Comment:
Did you test this out? I tried this, and ran into issues with headsets.

I could use some, but not all channels.

You'll have to test this with the pre-spawned (lockers, etc.) ones that start on your person, and ones that are created after round start.